docs/sources/setup-grafana/configure-security/configure-authentication/github.md

```diff
@@ -101,6 +101,33 @@ allow_sign_up = true
 allowed_organizations = github google
 ```
 
+### Map roles
+
+You can use GitHub OAuth to map roles. During mapping, Grafana checks for the presence of a role using the [JMESPath](http://jmespath.org/examples.html) specified via the `role_attribute_path` configuration option.
+
+For the path lookup, Grafana uses JSON obtained from querying GitHub's API [`/api/user`](https://docs.github.com/en/rest/users/users#get-the-authenticated-user=) endpoint and a `groups` key containing all of the user's teams. The result of evaluating the `role_attribute_path` JMESPath expression must be a valid Grafana role, for example, `Viewer`, `Editor` or `Admin`. For more information about roles and permissions in Grafana, refer to [Roles and permissions]({{< relref "../../../administration/roles-and-permissions/" >}}).
```

Comment:
nit: I'd suggest to remove this part "and a `groups` key containing all of the user's teams", as the `group` is a bit confusing. I tried to figure out what is this groups and where it's coming from but was not able to find anything - I assume I am looking at the wrong place, however looking at the code, we just [fetch the teams](https://github.com/grafana/grafana/blob/acd85314b34bf467fbd7b4fe1d0e356ca68a1cbb/pkg/login/social/github_oauth.go#L111) and filter from there. If this is the case, I think we could just mention it.